packages/orchestration/src/cosmos-api.ts

```diff
@@ -39,23 +38,26 @@ export type CosmosValidatorAddress = ChainAddress & {
 
 /** Represents an IBC Connection between two chains, which can contain multiple Channels. */
 export type IBCConnectionInfo = {
-  id: IBCConnectionID; // e.g. connection-0
+  // XXX really IBCConnectionID but our chain info fetcher doesn't know
```

Comment:
Why can't we teach it?

Reply:
I didn't say we can't. The reason I didn't is that it would require a TypeScript AST generator. We're using `JSON.stringify` which does the job.